rider/urls.py

```diff
@@ -9,16 +9,16 @@
                        url(r'^register_push/$',
                            views.RiderPushUpdateAPI.as_view()),
 					# get affinity group data
-					url(r'^list_group/(?P<r_id>\d+)/$', 
+					url(r'^list_group/(?P<r_id>[^/]+)/$', 
```

Comment:
Why are you adding a trailing slash to all the URLs? I have no issue with this, I'm just curious.